docs/quick-start.md

```diff
@@ -148,3 +148,41 @@ plt.show()
 The environmental noise in tstrait follows a normal distribution. Please see [](phenotype_model)
 for mathematical details on the phenotype model and [](effect_size_dist) for details on
 specifying the effect size distribution.
+
+(normalise_phenotype)=
+
+## Normalise Phenotype
+
+The simulated phenotypes can be scaled by using the {func}`normalise_phenotypes` function. The function
+will first normalise the phenotype by subtrating the mean of the input phenotype from each
```

Comment:
typo, subtracting

Reply:
I just fixed it now, and thank you for spotting this